### PR TITLE
 #167484609 Enable user to like and dislike a comment

### PR DESCRIPTION
**Pull request by Jkadhuwa**
### What does this PR do?
Enable user to like and dislike a comment

### How should this be manually tested?
- `git fetch`
- `git checkout ft-user-like-specific-comment-167484609`

- `npm install`

- `npm run dev`

-  Using postman test using the following endpoints
    ```
    post /api/v1/comments/:id/like
       - To like a specific comment

    post /api/v1/comments/:id/dislike
       - To dislike a comment

### What are the relevant pivotal tracker stories?
[Like or Dislike a comment](https://www.pivotaltracker.com/story/show/167484609)

### Screenshots
<img width="754" alt="Screen Shot 2019-08-29 at 14 09 33" src="https://user-images.githubusercontent.com/21122345/63935738-c90c1d00-ca66-11e9-80ee-cf2c5f3f12c5.png">
<img width="789" alt="Screen Shot 2019-08-29 at 14 10 05" src="https://user-images.githubusercontent.com/21122345/63935775-dcb78380-ca66-11e9-895a-dfa83f7986e1.png">
<img width="833" alt="Screen Shot 2019-08-29 at 14 10 22" src="https://user-images.githubusercontent.com/21122345/63935781-de814700-ca66-11e9-83f3-3e6852a555e2.png">

### CODE REVIEW

**Review by harerakalex — CHANGES_REQUESTED** [review]
@Jkadhuwa, In `articleController` test has gone down. please increase this test otherwise It looks good
Screen Shot 2019-08-29 at 13 42 35

**Comment by Jkadhuwa** [comment]
> @Jkadhuwa, In `articleController` test has gone down. please increase this test
> Screen Shot 2019-08-29 at 13 42 35

@harerakalex in the `articleController` these are pagination lines added by @Malaba6

File: src/tests/likeDislikeComment.test.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+        expect(res).have.status(404);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message');
+        expect(res.body.message).to.deep.equal(
+          'Sorry! Comment not found.'
+        );
+        done();
+      });
+  });
+  it('Should a message of success when a like is registered', (done) => {
```
Comment (malaba6) [inline src/tests/likeDislikeComment.test.js]: `should a message` maybe you meant `should return a mes...`

File: src/tests/likeDislikeComment.test.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message');
+        expect(res.body.message).to.deep.equal(
+          'You have successfully removed your dislike for this comment.'
+        );
+        done();
+      });
+  });
+  it('Should retrieve comments of the article with correct number of dislikes', (done) => {
```
Comment (malaba6) [inline src/tests/likeDislikeComment.test.js]: This message is misleading, looking at it, I expect to see the number of `dislikes`, but actually what you display here is the number of `likes`

**Review by malaba6 — APPROVED** [review]

File: src/tests/likeDislikeComment.test.js (hunk 1 above)
Comment (Jkadhuwa) [reply src/tests/likeDislikeComment.test.js]: done

File: src/tests/likeDislikeComment.test.js (hunk 2 above)
Comment (Jkadhuwa) [reply src/tests/likeDislikeComment.test.js]: Thanks @Malaba6, done

**Review by raymond42 — APPROVED** [review]
I've tested this endpoint and is working fine as intended.

**Pull request merged**